packages/@lwc/perf-benchmarks/scripts/build.js

```diff
@@ -12,8 +12,8 @@
 const path = require('path');
 const fs = require('fs');
 const { promisify } = require('util');
-const glob = promisify(require('glob'));
-const globHash = require('glob-hash');
+const { glob } = promisify(require('glob'));
```

Comment:
```suggestion
const { glob } = require('glob');
```

The function signature for `glob` returns a promise, we can skip wrapping it in `promisify`.